CHANGELOG.md

```diff
@@ -3,6 +3,723 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 8.0.0-alpha.0
+
+<!-- generated comparing v7.22.1..master -->
+
+_Nov 7, 2024_
+
+We'd like to offer a big thanks to the 21 contributors who made this release possible. Here are some highlights ✨:
+
+TODO INSERT HIGHLIGHTS
+
+From https://github.com/mui/mui-x/pull/14913
+
+### Tree View
```

Comment:
I thought the `## Changelog` ordered/categorised the changelogs. I was wrong 🙃 

This output is very confusing

Reply:
Nop it just adds everything at the top :laughing: 
It would be a nice improvement

Reply:
We should move them in their respective product line inside a `#### Breaking changes` before merging this PR :+1:

Reply:
moved